pkg/cloudprovider/aws/packing/packing.go

```diff
@@ -78,6 +80,42 @@ func (p *podPacker) Pack(ctx context.Context, pods []*v1.Pod) ([]*Packing, error
 
 // TODO filter instance types based on node contraints like availability zones etc.
 func (p *podPacker) getNodeCapacities() []*nodeCapacity {
+	nodeCapacities := []*nodeCapacity{}
+	describeInstanceTypesInput := &ec2.DescribeInstanceTypesInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("processor-info.supported-architecture"),
+				Values: []*string{aws.String("x86_64")},
+			},
+			{
+				Name:   aws.String("supported-usage-class"),
+				Values: []*string{aws.String("on-demand")},
+			},
+			{
+				Name:   aws.String("supported-virtualization-type"),
+				Values: []*string{aws.String("hvm")},
+			},
+		},
+	}
+	err := p.ec2.DescribeInstanceTypesPagesWithContext(context.TODO(), describeInstanceTypesInput, func(page *ec2.DescribeInstanceTypesOutput, lastPage bool) bool {
+		for _, instanceTypeInfo := range page.InstanceTypes {
+			nodeCapacity, err := instanceTypeInfoToNodeCapacity(*instanceTypeInfo)
+			if err != nil {
+				zap.S().Warnf("Failed to convert instanceTypeInfo to a nodeCapacity, %s", err.Error())
+				continue
+			}
+			nodeCapacities = append(nodeCapacities, nodeCapacity)
+		}
+		return lastPage
+	})
+
+	if err != nil {
+		zap.S().Warnf("Failed to fetch instance types using ec2.DescribeInstanceTypes, %s", err.Error())
+		return fallbackNodeCapacities
```

Comment:
I think we should remove this - I don't think there's a reasonable fallback in this case (and in general fallback is a bad idea, see https://aws.amazon.com/builders-library/avoiding-fallback-in-distributed-systems/ for more rationale).